packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -51,6 +51,17 @@ export interface VideoGalleryProps {
   onDisposeRemoteStreamView?: (userId: string) => Promise<void>;
   /** Callback to render a particpant avatar */
   onRenderAvatar?: (props: PlaceholderProps, defaultOnRender: (props: PlaceholderProps) => JSX.Element) => JSX.Element;
+
+  /**
+   * Whether to display the user's name in video tile.
+   * @defaultValue `true`
+   */
+  showDisplayName?: boolean;
+  /**
+   * Whether to display a mute icon beside the user's display name.
```

Comment:
What happens if `showDisplayName` is false but `showMuteIndicator` is true?

Reply:
Its optional in the SDK but I am starting to question if it should be optional in the UI components.  This would be a good question to  raise if there is a use case for having no name? 

Silly idea. Perhaps we use the acs id if the no display name is set? The only way for someone to go out of their way to have no name is to explicitly apply "" to the displayName prop?

Reply:
Removed showDisplayName option.
But if no display name is available, only a mute icon will be displayed.